region/crateria/central/Flyway.json

```diff
@@ -136,6 +136,57 @@
         "The aim-down method requires specific positioning to ensure Samus barely clears the door ledge with a frame-perfect jump."
       ]
     },
+    {
+      "link": [1, 1],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        {"not": "f_ZebesSetAblaze"},
+        {"or": [
+          "h_CrystalFlash",
+          {"and": [
+            {"resourceMissingAtMost": [{"type": "Missile", "count": 0}]},
+            {"or": [
+              {"and": [
+                {"resourceAvailable": [{"type": "RegularEnergy", "count": 99}]},
+                {"disableEquipment": "ETank"},
+                {"partialRefill": {"type": "ReserveEnergy", "limit": 100}}
+              ]},
+              {"and": [
+                {"resourceAvailable": [{"type": "RegularEnergy", "count": 19}]},
```

Comment:
I think entering with low energy and leaving with "high" energy can be the base case, no weapon.